client/src/pages/AuthPage.js

```diff
@@ -63,7 +63,7 @@ function AuthPage({ classes, ...rest }) {
 
   return (
     <PageContainer className="pageContainer">
-      <Navbar>
+      <Navbar transparent={true }>
```

Comment:
This is equivalent to: ` <Navbar transparent>` , it automatically takes a `true` value. Matter of choice though

Reply:
I think briefer is better here, thanks for catching that